lib/game.rb

```diff
@@ -1 +1,22 @@
-puts "Welcome to Tic-Tac-Toe!"
\ No newline at end of file
+class Game
+    def initialize
+        puts "Welcome to Tic-Tac-Toe!"
+    end
```

Comment:
Do you think it's odd to have this welcome message in the initialize method?  I think of initialize as holding state.